vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -143,14 +144,27 @@ func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp
 		kec.ReplicasEnvVarsTransform(manifest.Client),
 	}
 	extra = append(extra, r.extension.Transformers(instance)...)
-	return common.Transform(ctx, manifest, instance, extra...)
+	return common.Transform(ctx, manifest, instance)
```

Comment:
@pierDipi I think here we missed `extra` correct?

Reply:
I'm not sure why it was removed

Reply:
I'm adding it back https://github.com/knative/operator/pull/1587

Reply:
Thanks for the good catch!